deploy/template.yaml

```diff
@@ -28,6 +28,21 @@ Parameters:
     Description: "The ARN of the permissions boundary to apply when creating IAM roles"
     Type: String
     Default: "none"
+  CodeSigningConfigArn:
+    Type: String
+    Description: Asserts that lambdas are signed when deployed.
+    Default: "none"
+  DeploymentStrategy:
+    Description: "Predefined deployment configuration for ECS application"
+    Type: String
+    Default: "None"
```

Comment:
Why not "none" just for being consistent.

Reply:
because dev platform have used None - this is a dev platform choice, not ours

Reply:
I would love to use none ;) personally, but they defaulted to using None and there are a bunch of other teams already using this, so too late to change this now.